src/App/App.js

```diff
@@ -7,15 +7,27 @@ class App extends Component {
   constructor() {
     super()
     this.state = {
-
+      movies: [],
+      error: ''
     }
   }
 
-  render = () => {
+  componentDidMount() {
```

Comment:
When we get into testing, this will be moved to its own file (one that holds all fetch requests!) But it's working great!